services/venice-server/src/test/java/com/linkedin/venice/stats/AggVersionedDIVStatsTest.java

```diff
@@ -108,43 +108,24 @@ public void testStatsCanUpdateVersionStatus() {
     Assert.assertEquals(reporter.query("." + storeName + "--future_version.Gauge").value(), 1d);
 
     long consumerTimestampMs = System.currentTimeMillis();
-    double v1ProducerBrokerLatencyMs = 801d;
 
     double v1ProducerToSourceBrokerLatencyMs = 811d;
     double v1SourceBrokerToLeaderConsumerLatencyMs = 211d;
-    double v1ProducerToLeaderConsumerLatencyMs = 1011d;
     stats.recordLeaderLatencies(
         storeName,
         1,
         consumerTimestampMs,
         v1ProducerToSourceBrokerLatencyMs,
-        v1SourceBrokerToLeaderConsumerLatencyMs,
```

Comment:
Similar here, any verification for the newly added `leader_div_completion_latency`?

Reply:
please see my reply above